tests/accounts/test_accounts_profile.py

```diff
@@ -149,10 +149,89 @@ def test_profile_login_using_facebook(accounts_base_url, selenium):
 
 @test_case('C195557')
 @nondestructive
-@expected_failure
 @accounts
 def test_admin_pop_up_console(accounts_base_url, selenium):
     """Test the pop up console."""
+    page = Profile(selenium, accounts_base_url).open()
+    assert(not page.logged_in), 'Active user session unexpected'
+    username = os.getenv('ADMIN_USER')
+    password = os.getenv('ADMIN_PASSWORD')
+    page.log_in(username, password)
+    assert(page.logged_in), 'User "{0}" not logged in'.format(username)
+    assert(page.is_admin), 'User is not an administrator'
+    assert(page.has_username), 'No username found'
+    popup = page.open_popup_console()
+    from time import sleep
```

Comment:
We shouldn't have inline imports in the tests; if it needs to wait for the console to open, that should be in the model.